diff --git a/node_modules/@timvir/blocks/Code/index.js b/node_modules/@timvir/blocks/Code/index.js
index ada435e..f4a2c95 100644
--- a/node_modules/@timvir/blocks/Code/index.js
+++ b/node_modules/@timvir/blocks/Code/index.js
@@ -1,5 +1,5 @@
 import Highlight, { defaultProps } from 'prism-react-renderer';
-import theme from 'prism-react-renderer/themes/github';
+import theme from 'prism-react-renderer/themes/github/index.cjs.js';
 import * as React from 'react';
 import * as Icons from 'react-feather';
 import { useImmer } from 'use-immer';
